ui/app/src/modules/Content/Dependencies/DependenciesDialogUI.tsx

```diff
@@ -0,0 +1,473 @@
+    open,
+    handleClose,
+    isEditableItem,
+    assetsTypes,
+    editDialogConfig,
+    setEditDialogConfig,
+    handleEditorDisplay
+  } = props;
+  const classes = dependenciesDialogStyles({});
+  const { formatMessage } = useIntl();
+
+  return (
+    <Dialog
+      onClose={handleClose}
+      open={open}
+      fullWidth={true}
+      maxWidth={'md'}
+      classes={{
+        root: classes.root,
+        paper: classes.dialogPaper
+      }}
+    >
+      {
+        <>
+          <DialogHeader
+            title={formatMessage(translations.headerTitle)}
+            onClose={handleClose}
+          />
+          <DialogBody>
+            <div className={classes.selectionContent}>
+              <Chip
+                variant="outlined"
+                deleteIcon={isEditableItem(state.item.uri) ? <CreateIcon/> : null}
+                onDelete={isEditableItem(state.item.uri) ?
+                  () => {
+                    handleEditorDisplay(state.item);
+                  } :
+                  null
+                }
+                classes={{
+                  root: classes.selectedItem,
+                  label: classes.selectedItemLabel,
+                  deleteIcon: classes.selectedItemEditIcon
+                }}
+                label={
+                  <>
+                    <span className='label'>Selected</span>
+                    <InsertDriveFileOutlinedIcon className='item-icon'/>
+                    <span className='item-title'>{state.item.internalName}</span>
+                  </>
+                }
+              />
+
+              <FormControl className={classes.formControl}>
+                <Select
+                  value={state.selectedOption}
+                  onChange={(event: React.ChangeEvent<{ value: unknown }>) => {
+                    setState({ selectedOption: event.target.value as string });
+                  }}
+                  inputProps={{
+                    className: classes.select
+                  }}
+                >
+                  <MenuItem value='depends-on'>
+                    <FormattedMessage
+                      id="dependenciesDialog.dependsOn"
+                      defaultMessage="Items that depend on selected item"
+                    />
+                  </MenuItem>
+                  <MenuItem value='depends-on-me'>
+                    <FormattedMessage
+                      id="dependenciesDialog.dependsOnMe"
+                      defaultMessage="Dependencies of selected item"
+                    />
+                  </MenuItem>
+                </Select>
+              </FormControl>
+            </div>
+            <SuspenseWithEmptyState
+              resource={resource}
+              withEmptyStateProps={{
+                emptyStateProps: {
+                  title: (
+                    <FormattedMessage
+                      id="dependenciesDialog.emptyStateMessage"
+                      defaultMessage="No Dependencies found"
```

Comment:
This message should change depending on the type of dependencies shown.
- Dependencies of selected item: `"{itemName}" has no dependencies.`
- Items that depend on selected item: `Nothing depends on "{itemName}"`